rcsbsearchapi/search.py

```diff
@@ -964,7 +969,6 @@ class Session(Iterable[str]):
     return_type: ReturnType
     start: int
     rows: int
-    computational: bool
```

Comment:
Can `return_content_type: List[ReturnContentType]` also be added to the initialization here, or does doing so cause a problem?

Reply:
Also, can you change the `url` variable above to be `https` instead of `http`?

Reply:
Ideally, service URL should be part of the client configuration (not in the client code)

Reply:
Good point @valasatava, thanks for the tip! In that case, @rushambhatt, if you can place the definition of the url in the `consts.py` file, that would be appropriate. (Maybe call it `RCSB_SEARCH_API_QUERY_URL`?)

Reply:
Will do. Thanks!

Reply:
Having a default URL is fine. However, to provide for a greater flexibility there should be a mechanism to provide the URL externally and override the default. Image, you use this client in your application and need to connect to a testing Search API server.